rotkehlchen/chain/ethereum/decoding/decoder.py

```diff
@@ -96,26 +89,6 @@ def _maybe_enrich_transfers(
                     event.extra_data = {AIRDROP_IDENTIFIER_KEY: '1inch'}
             return DEFAULT_DECODING_OUTPUT
 
-        if tx_log.topics[0] == GNOSIS_CHAIN_BRIDGE_RECEIVE and tx_log.address == '0x88ad09518695c6c3712AC10a214bE5109a655671':  # noqa: E501
```

Comment:
why is this code removed? Where did it go? Is it covered by the new decoder? I see `0x88ad09518695c6c3712AC10a214bE5109a655671` is the omnibridge?